bot/cogs/moderation/utils.py

```diff
@@ -97,8 +97,13 @@ async def post_infraction(
                 return
 
 
-async def has_active_infraction(ctx: Context, user: UserSnowflake, infr_type: str) -> bool:
-    """Checks if a user already has an active infraction of the given type."""
+async def get_active_infractions(
+        ctx: Context,
+        user: UserSnowflake,
+        infr_type: str,
+        send_msg: bool = True
```

Comment:
Please document this new parameter in the docstring. It can just be put on a separate line

Reply:
Addressed in most recent commit